client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -19,121 +23,128 @@ import CompactFormToggle from 'components/forms/form-toggle/compact';
 import FormInputValidation from 'components/forms/form-input-validation';
 import UploadImage from 'blocks/upload-image';
 
-class ProductForm extends Component {
-	handleFieldChange = ( { currentTarget: { name, value } } ) => {
-		this.props.onFieldChange( name, value );
-	};
+const REDUX_FORM_NAME = 'simplePaymentsForm';
+
+// Export some selectors that are needed by the code that submits the form
+export const getProductFormValues = state => getFormValues( REDUX_FORM_NAME )( state );
+export const isProductFormValid = state => isValid( REDUX_FORM_NAME )( state );
+export const isProductFormDirty = state => isDirty( REDUX_FORM_NAME )( state );
+
+// Validation function for the form
+const validate = ( values, props ) => {
+	// The translate function was passed as a prop to the `reduxForm()` wrapped component
+	const { translate } = props;
+	const errors = {};
+
+	if ( ! values.title ) {
+		errors.title = translate( 'Product name can not be empty.' );
+	}
+
+	if ( ! values.price ) {
+		errors.price = translate( 'Price can not be empty.' );
```

Comment:
The 'can not' is being used already on the `/me/account` form when validating an email address:
<img width="459" alt="me-account-email" src="https://user-images.githubusercontent.com/664258/29071023-1466245a-7c42-11e7-82dc-49449bfd3ed3.png">
I copied these messages verbatim into this form, too, and adapted other ones in the product form to the same spelling.

The "Product name cannot be empty" message is coming from the previous version of the `SimplePaymentsDialog`, which is still a work in progress. If we reach agreement on this, I'd rather update the spelling of the existing message.

Maybe @yoavf has some advice on how to update a spelling of an English message and keeping all the translations intact. Is there a way how to just fix a typo or alternative spelling like this? I'm not terribly familiar with the i18n machinery and GlotPress...